library/Fission/Internal/UTF8.hs

```diff
@@ -55,6 +57,10 @@ stripN n = Text.dropEnd i . Text.drop i
     i :: Int
     i = fromIntegral n
 
+-- | Helper for showing nytestring without quotes
+stripQuotes :: Char8.ByteString -> [Char]
```

Comment:
This is probably fine as is. I would either
- rename this something that follows the pattern `stripN...` since this doesn't actually "strip quotes" it strips one char on each side
- or since this function is only used once, just compose functions in `Fission.Web.Types` instead of writing a new function